### ISSUE TITLE
Unexpected token process.env.NODE_ENV

### ISSUE THREAD

**Issue by ShuCh3n** [bigcommerce/checkout-sdk-js#2138]
I'm currently using Vite, but during rollup im getting an unexpected token error. This is regarding the warning text in file create-checkout-service.ts on line 95.

```javascript
getDefaultLogger().warn(
    'Note that the development build is not optimized. To create a production build, set process.env.NODE_ENV to `production`.',
);
```

When I remove the  "process.env.NODE_ENV" text it will build just fine.

**Comment by bc-peng** [comment]
Could you kindly provide your question?

**Comment by animesh1987** [comment]
Hey @ShuCh3n are you building SDK locally for dev? Where are you removing the text from?

**Comment by ShuCh3n** [comment]
<img width="918" alt="Screenshot 2023-08-22 at 16 52 35" src="https://github.com/bigcommerce/checkout-sdk-js/assets/7081446/d256c886-3379-4255-ba50-381ad94073d7">

@animesh1987 It Doesn't matter for which build, either dev or production both are giving me this error. I suspect that Vite replaces the process.env.NODE_ENV. So when I remove this part of text in node_modules/@bigcommerce/checkout-sdk/dist/checkout-sdk.js it will build just fine.
